.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+jobs:
+  build-wheels:
+    name: upload pybamm-cookie wheel artifacts
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout pybamm-cookie
+      - uses: actions/checkout@v4
+
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.12
```

Comment:
```suggestion
```

We'll need to keep updating the Python version, so we might as well remove it and let it use the latest available one